src/practices/Java/JavaDependenciesVersionMajorLevel.ts

```diff
@@ -55,7 +56,7 @@ export class JavaDependenciesVersionMajorLevel extends PracticeBase {
     return latestVersionsJson;
   }
 
-  setData(pkgsToUpdate: PkgToUpdate[]): void {
+  setData(pkgsToUpdate: UpdatedDependencyDto[]): void {
     this.data.details = [{ type: ReportDetailType.table, headers: ['Name', 'New', 'Current'], data: pkgsToUpdate }];
```

Comment:
Missing major statistics data?

Reply:
It sends data both from Major Level and Minor and from Patch Level as the method setData() is called both in JavaDependenciesVersionMajorLevel, and in JavaDependenciesVersionMinorPatchLevel